static/app/components/charts/onDemandMetricRequest.tsx

```diff
@@ -59,7 +59,7 @@ export class OnDemandMetricRequest extends EventsRequest {
         onError?.(errorMessage);
         this.setState({
           errored: true,
-          errorMessage,
+          errorMessage: 'Error fetching estimated data',
```

Comment:
Let's translate this. In a follow up PR should be fine in case we need to deploy this and don't want to wait for CI again.
```suggestion
          errorMessage: t('Error fetching estimated data'),
```